### DISCUSSION TITLE
[Greenfield API] invoice checkout.redirectAutomatically

### DISCUSSION THREAD

**Discussion by AryanJ-NYC** [btcpayserver/btcpayserver#2298] [Ideas / Feature Requests]
Legacy BTCPay invoices expose a boolean option when creating an invoice to `redirectAutomatically` which, when true, redirects a user automatically to the `redirectURL` after payment. That would be super helpful in the new Greenfield API.

**Comment by Kukks** [comment]
@pavlenex  Any idea how we can signal that I will pick this up and do it for the next release?

**Reply by ghost** [reply]
Things like labels & assignments don't exist in discussions...

**Reply by AryanJ-NYC** [reply]
If it makes things easier, I can go ahead and write up an issue (crazy that GH doesn't have this feature but I digress).

**Reply by bolatovumar** [reply]
I guess the way to signal that this is being worked on could be a comment that links to a new issue and we can then select this comment as an answer to this question so it's visible?

**Reply by pavlenex** [reply]
Discussions are still in alpha. In the future, there should be a way to convert discussion into an issue, which is what we can do. Until now just a new issue linking to here, or vice-versa should be enough.

**Comment by Kukks** [comment]
Done in #2321
